cluster-sim/cluster_sim/simulator.py

```diff
@@ -243,10 +243,12 @@ def setup(self, server_count, worker_count, volume_count, nid_count, cluster_siz
         self.state['packages'] = {
             'server': {
                 'lustre': (0, '2.1.4', '1', 'x86_64'),
-                'lustre-modules': (0, '2.1.4', '1', 'x86_64'),
-                'lustre-osd-ldiskfs': (0, '2.1.4', '1', 'x86_64'),
-                'lustre-osd-zfs': (0, '2.1.4', '1', 'x86_64'),
-                'kernel-devel-3.10.0-514.10.2.el7_lustre': (0, '2.6.32', '1', 'x86_64'),
+                'lustre-dkms': (0, '2.1.4', '1', 'x86_64'),
+                'kmod-lustre-osd-ldiskfs': (0, '2.1.4', '1', 'x86_64'),
+                'lustre-osd-zfs-mount': (0, '2.1.4', '1', 'x86_64'),
+                'kernel-[0-9]*_lustre': (0, '2.6.32', '1', 'x86_64'),
```

Comment:
The glob is not expected to expand in the simulator.  It's just meant to match what is in the profile.

Not saying it's not a source of trouble, but if it is, that's a simulator problem, not a profile problem.